pycode/examples/epidata/PlotCaseData.py

```diff
@@ -1,7 +1,7 @@
 #############################################################################
 # Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
 #
-# Authors: 
+# Authors:
```

Comment:
I do not think that this file was compiling beforehand. Some things are outdated since there is no unittest for this file. I think fixing this is a different issue. I just wanted to introduce the new names here but I'm not sure anymore whether this is a good idea if this file currently does not compile.

Reply:
The same holds for PlotGermanData.py